### ISSUE TITLE
test_gre_profile:test_vxlan_profile in test_network.py KeyError

### ISSUE THREAD

**Issue by ssorenso** [F5Networks/f5-openstack-agent#845]
* Title: test_gre_profile:test_vxlan_profile in test_network.py:125 KeyError
* Attachments:
* Details:
```
    @log_test_call
    def test_vxlan_profile(setup):
        te = setup
        name = 'test_vxlan_profile'
        profile = network_helper.create_vxlan_multipoint_profile(te.mgmt_root, name,
                                                                 te.partition)
        add_resource_teardown(te.request, profile)
        params = {'params': {'$filter': 'partition eq %s' % te.partition}}
        vc = te.mgmt_root.tm.net.tunnels.vxlans
        profiles = vc.get_collection(requests_params=params)
        profile_names = (r.name for r in profiles)
        assert(name in profile_names)
        p = te.mgmt_root.tm.net.tunnels.vxlans.vxlan
        p.load(name=name, partition=te.partition)
        payload = NetworkHelper.vxlan_multipoint_profile_defaults
        for k in payload.keys():
            if k == 'partition':
                continue
>           assert(p.__dict__[k] == profile.__dict__[k])
E           KeyError: 'floodingType'

test_network.py:125: KeyError
```

##### Associated md5sums:

* 221a2db7c3577bceab793f30d3229632

##### Impacted tests:
* `openstack_agent_mitaka_12.1.1-overcloud.87.consoleText.test_vxlan_profile`
* `openstack_agent_newton_12.1.1-overcloud.64.consoleText.test_vxlan_profile`

#### OpenStack Release
* mitaka
* newton

#### Description
<Describe the enhancement request in detail>

* Details: For bugs, fill out the template below.

#### Agent Version
* stable/newton - `d884f8e70e84a07d5d65161df71c270cc847aefa`
* mitaka - `6d48f81c945da2e783f9c74a4c353dd52adee9b4`

#### Operating System
Nightly

**Comment by ssorenso** [comment]
Additional sum: 221a2db7c3577bceab793f30d3229632

**Comment by ssorenso** [comment]
@dflanigan, this is an actual implementation error, not test error.  The scenario expectation has changed for a given set of input that is not time dependent here.

Neutronless tests (agent tests) are time irrelevant as they are in serial and has nothing that is in parallel due to the fact that the agent is only talking to the BIG-IP with no intermediate dependencies.

**Comment by ssorenso** [comment]
Additional sums from today:
md5sum:cc03f4d3440de6eb9ac082e3f2dde16f
* `openstack_agent_mitaka_12.1.2-overcloud.5.consoleText`
  * Type: failures
  * Test: `test_vxlan_profile`
* `openstack_agent_newton_12.1.2-overcloud.5.consoleText`
  * Type: failures
  * Test: `test_vxlan_profile`
* `openstack_agent_liberty_12.1.2-overcloud.5.consoleText`
  * Type: failures
  * Test: `test_vxlan_profile`

**Comment by richbrowne** [comment]
Fixed by: 0cdebc5be80dc4ba8759672f6ee808123c590e5e
